src/main/java/frc/robot/subsystems/vision/VisionEstimation.java

```diff
@@ -157,26 +173,29 @@ private Matrix<N3, N1> confidenceCalculator(EstimatedRobotPose estimation) {
                         / (1
                                 + ((estimation.targetsUsed.size() - 1) * VisionConstants.TAG_PRESENCE_WEIGHT)));
 
+        confidenceTelemetry.addNumber(confidenceMultiplier);
```

Comment:
Adding the current confidence multiplier to this `MeanPerPeriodTelemetry` helper, which will send the mean (avg) conf. mult. every half sec.